prepare.icons.js

```diff
@@ -1,4 +1,9 @@
 import { copyIcons } from "@vueless/plugin-vite/utils/iconLoader.js";
 
 /* Copy SVG icons from the default icon library into the assets' folder. */
-copyIcons("vuelessIcons", "vueless");
+await copyIcons({
+  mode: "vuelessIcons",
+  env: "vueless",
+  debug: false,
+  targetFiles: [process.cwd()],
```

Comment:
this is not the right way, I guess. Please use the variant from the main branch.